tests/test_cli.py

```diff
@@ -365,26 +443,26 @@ def test_log_flags(self, caplog, gateway):
         assert '"log_level": "NOTICE"' in caplog.text
         assert '"log_print_level": "INFO"' in caplog.text
         caplog.clear()
-        cli(["set_spdk_nvmf_logs", "-f"])
-        assert "Set SPDK nvmf logs : True" in caplog.text
+        cli(["set_spdk_nvmf_logs"])
+        assert "Set SPDK nvmf logs: True" in caplog.text
         caplog.clear()
         cli(["get_spdk_nvmf_log_flags_and_level"])
         assert '"nvmf": true' in caplog.text
         assert '"nvmf_tcp": true' in caplog.text
         assert '"log_level": "NOTICE"' in caplog.text
         assert '"log_print_level": "INFO"' in caplog.text
         caplog.clear()
-        cli(["set_spdk_nvmf_logs", "-f", "-l", "DEBUG"])
-        assert "Set SPDK nvmf logs : True" in caplog.text
+        cli(["set_spdk_nvmf_logs", "-l", "DebuG"])
```

Comment:
better to use a variable instead of a string (there are many repeatedly used strings..)

Reply:
It didn't seem important enough to change the entire code. This code was already there, I just changed the case of DEBUG to DebuG